backend/src/models.ts

```diff
@@ -0,0 +1,135 @@
+export interface LatLng {
+  lat: number;
+  lng: number;
+}
+
+export interface LatLon {
+  lat: number;
+  lon: number;
+}
+
+export interface PointData extends LatLng {
+  properties?: PointProperties;
+  value?: number;
+  metadata?: any;
+}
+
+export type Path = PointData[];
+
+export interface Bounds {
```

Comment:
Some functions here which I think are for the withBounds/BoundedMap func (so potentially not in use)